newapi/ooniapi/measurements.py

```diff
@@ -865,6 +1031,8 @@ def list_measurements():
     # Assemble the "external" query. Run a final order by followed by limit and
     # offset
     query = fp_query.offset(offset).limit(limit)
+    query_params["param_1"] = limit
+    query_params["param_2"] = offset
```

Comment:
Where are these being used?

Reply:
IIRC it was required by offset(...) and limit(...) but I'm not focusing on PG code in this PR.